internal/server/suite_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"sigs.k8s.io/cluster-api/util"
 	ctrl "sigs.k8s.io/controller-runtime"
 
+	"github.com/projectsveltos/addon-controller/api/v1alpha1"
```

Comment:
nit: can you please change to 

configv1alpha1 "github.com/projectsveltos/addon-controller/api/v1alpha1"

Nothing wrong here, it is just that throughout all Sveltos repos we use

configv1alpha1 "github.com/projectsveltos/addon-controller/api/v1alpha1"
libsveltosv1alpha1 "github.com/projectsveltos/libsveltos/api/v1alpha1"